build.gradle

```diff
@@ -1439,7 +1440,7 @@ project(':streams:streams-scala') {
   }
 
   jar {
-    dependsOn 'copyDependantLibs'
+    dependsOn 'copyDependantLibs' , 'scaladocJar'
```

Comment:
Hmm, seems weird to require scaladoc when building jars.

Reply:
@ijuma good point, I've removed it and still get the required doc jars